src/sass/_theme_rst.sass

```diff
@@ -253,9 +253,13 @@
     line-height: 0
     font-size: 90%
 
-  // Horizontal lists have multiple columns; use full width
   .hlist
     width: 100%
+    tr
+      display: flex
+      flex-flow: row wrap
+    td
+      margin-right: auto
```

Comment:
Currently, this theme does not use flex box layout because we have not officially dropped support for IE11, or IE10 for that matter. There is limited support for flex box in IE10+11, but you will need to add more rules.

This PR does break hlists on IE11:

![image](https://user-images.githubusercontent.com/1140183/113434770-ce49f000-939e-11eb-82d5-9bb216ad3ee1.png)

This is what IE11 looks like on latest release, for comparison:

![image](https://user-images.githubusercontent.com/1140183/113434811-dbff7580-939e-11eb-972e-d3015cba98ee.png)